src/local_scheduler/local_scheduler.cc

```diff
@@ -1195,6 +1199,11 @@ void handle_task_scheduled_callback(Task *original_task,
   TaskExecutionSpec *execution_spec = Task_task_execution_spec(original_task);
   TaskSpec *spec = execution_spec->Spec();
 
+  LOG_INFO("[LS][handle_task_scheduled] task scheduled on LS with spillback %d\n",
+           execution_spec->getSpillbackCount());
+  /* Set the tasks's local scheduler entrypoint time. */
+  execution_spec->setLastTimeStamp(current_time_ms());
```

Comment:
same question here, it's a lot of system calls per task

Reply:
what is a lot of system calls per task? We need to timestamp the task's entrypoint into the local scheduler for the spillback mechanism to work. I'll remove the timestamp in the TaskExecutionSpec ctor and leave it at the two local scheduler entrypoints.